docs/appendix/apis/components/arm.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Arm API"
+linkTitle: "Arm"
+weight: 20
+type: "docs"
+description: "The arm API allows you to give commands to your arm components with code instead of with the graphical interface of the Viam app"
+icon: true
+images: ["/icons/components/arm.svg"]
+---
+
+The arm API allows you to give commands to your [arm components](/components/arm/) with code instead of with the graphical interface of the [Viam app](https://app.viam.com/).
+
+The arm component supports the following methods:
+
+{{< readfile "/static/include/components/apis/generated/arm-table.md" >}}
+
+## Establish a connection
+
+To get started using Viam's SDKs to connect to and control your machine, go to your machine's page on [the Viam app](https://app.viam.com), navigate to the **CONNECT** tab's **Code sample** page, select your preferred programming language, and copy the sample code.
+
+{{% snippet "show-secret.md" %}}
+
+When executed, this sample code will create a connection to your machine as a client.
+Then control your machine programmatically by adding API method calls as shown in the following examples.
```

Comment:
This feels a bit unnecessary, like if someone knows what an API is, they don't need this sentence.

Reply:
```suggestion
```